PHP/PHP Source.sublime-syntax

```diff
@@ -560,6 +561,55 @@ contexts:
         - include: identifier-constant-pop
     - include: use-statement-common
     - include: identifier-constant-pop
+  enum:
+    - match: '(?i)\b(enum)\s+({{identifier}})\s*'
+      captures:
+        1: storage.type.enum.php
```

Comment:
Maybe `keyword.declaration.enum.php` is more suitable than giving it a `storage` scope to align with https://github.com/sublimehq/Packages/issues/582 ?

Reply:
Agree. Currently `class` (`storage.type.php`), `interface` (`storage.type.php`) and `trait` (`storage.type.trait.php`) are `storage`.

Reply:
Yes, hence the suggestion to keep `storage` for backwards compatibility and also use `keyword.declaration` as a forward fill as described in https://github.com/sublimehq/Packages/issues/582#issuecomment-453231444

Reply:
Resolved in https://github.com/sublimehq/Packages/pull/2822/commits/2a26d837e5a4ad385879573a035638ab68eed21e